pelican/plugins/_utils.py

```diff
@@ -53,28 +53,27 @@ def load_legacy_plugin(plugin, plugin_paths):
         spec = importlib.util.find_spec(plugin)
     if spec is None:
         raise ImportError('Cannot import plugin `{}`'.format(plugin))
-    else:
-        # Avoid loading the same plugin twice
-        if spec.name in sys.modules:
-            return sys.modules[spec.name]
-        # create module object from spec
-        mod = importlib.util.module_from_spec(spec)
-        # place it into sys.modules cache
-        # necessary if module imports itself at some point (e.g. packages)
-        sys.modules[spec.name] = mod
+    # Avoid loading the same plugin twice
+    if spec.name in sys.modules:
+        return sys.modules[spec.name]
+    # create module object from spec
+    mod = importlib.util.module_from_spec(spec)
+    # place it into sys.modules cache
+    # necessary if module imports itself at some point (e.g. packages)
+    sys.modules[spec.name] = mod
+    try:
```

Comment:
While correct, this can be debatable. Sometimes it's better to explicitly (and visually) separate two branches. For example here with the changes `raise` block kind of blends in with the rest of the code. With an explicit `else` it is more clear that `raise` is also one of the "main" endpoints of the function.

Reply:
@yezz123 would you be willing to remove the mentioned part from the pr so we can get this merged?

Reply:
@yezz123: Hi Yasser. Last call… Would you please remove this specific change from your PR so we can get it merged?

Reply:
Hello @justinmayer @ingwinlu Sorry for the late reply i just fix it